src/commands/server/update.ts

```diff
@@ -149,22 +149,6 @@ export default class Update extends Command {
             cli.warn('Consider using \'chectl update [stable|next]\' to update to the latest version of chectl.')
           }
 
-          const cheCluster = await kubeHelper.getCheCluster(flags.chenamespace)
```

Comment:
We don't need this anymore?

Reply:
I think, user can update CR images using patch file, that's why I removed this warning section: https://github.com/che-incubator/chectl/pull/923/files#diff-bb7b8c94c8183998fc0e7c19ea87dd2bL152-L167

Reply:
I reverted this warning.